packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.69",
+  "version": "7.0.0-beta.4",
```

Comment:
Is this correct?

Reply:
Yes 😊: We need a `@mui/base` release with the new package layout because `@mui/lab` depends on it. Otherwise, `@mui/lab` will fail to import in Node (ESM module with a fake-ESM dependency).

Reply:
🤔  Same actually applies to joy-ui

Reply:
None of our packages (at least the core ones) depend on `@mui/joy`, so I think we aren't forced to do this unless we want to update Joy's package layout. I think we should leave it as is for now.